package.json

```diff
@@ -55,7 +55,6 @@
     },
     "prepare": {
       "dependencies": [
-        "clean",
```

Comment:
This should be fine as we usually use it only on Publish from what I understand where no cache exist

Reply:
NIT: next time please split the commit into 2, as in case of issues with `package` we will need to revert the network as well